app/repos/exemption.js

```diff
@@ -122,9 +122,9 @@ const handleOrder2023 = (registration, data) => {
  */
 const setDefaults = (registration, data) => {
   if (
-    (registration.cdo_expiry === null || registration.cdo_expiry === undefined) &&
-    (registration.cdo_issued === null || registration.cdo_issued === undefined) &&
-    (data.cdoIssued !== null && data.cdoIssued !== undefined)
+    (data.cdoExpiry === null || data.cdoExpiry === undefined) &&
+    (data.cdoIssued !== null && data.cdoIssued !== undefined) &&
+    (registration.previous('cdo_issued') === null || registration.previous('cdo_issued') === undefined)
```

Comment:
Yes, the previous() is a better way to get previous values, but we already have them at the beginning of this call. Happy to let this go but it does make this logic sequelize-specific i.e. if we changed the ORM we would have to identify the change necessary in the logic (not just the queries).